core/includes/file.inc

```diff
@@ -1705,6 +1712,13 @@ function file_save_upload($form_field_name, $validators = array(), $destination
   }
 }
 
+function file_get_default_extensions() {
```

Comment:
At first sight I liked the idea of having a function for that. (With doc block, though.)

However, isn't this info already available via config: `config_get('system.core', 'file_default_allowed_extensions');`?